### DISCUSSION TITLE
구글 검색 노출 시키려면 어떻게 해야하나요?

### DISCUSSION THREAD

**Discussion by dgd03146** [hyesungoh/comet-land#110] [Q&A] [answered]
## 구글 검색 노출
안녕하세요 만들어주신 블로그 정말 잘 사용하고 있습니다. 너무 이쁘네요! 구글에서 검색하였을때 검색 결과가 노출이 안되는것 같은데 혹시 다른 설정을 해주어야하나요? 제가 깃블로그에 대한 이해가 부족해서요ㅠ

**Comment by hyesungoh** [comment, marked as answer]
안녕하세요! 잘 사용하고 계시다니, 기분이 좋습니다.

[google search console](https://search.google.com/search-console/about)에 등록해주신 다음에, `[블로그 url]/sitemap.xml`을 사이트맵에 제출해주시면 됩니다.

그럼 크롤링 후 지표 확인하실 수 있으십니다. (보톰 며칠 ~ 몇주 정도 걸립니다.)

---

google search console에 직접 등록하실 수도 있지만, 

`comet-land` 설정에 GA ID를 기입하셔서 GA를 통해 등록하시면 더욱 쉽게 등록하실 수 있습니다.

---

사용하시는 블로그 혹은 이력서를 사용 예로 [`comet-land readme`](https://github.com/hyesungoh/comet-land#use-case)에 올리고 싶으시다면 PR 부탁드리겠습니다.

좋은 하루되세요 감사합니다 :D

**Reply by dgd03146** [reply]
안녕하세요! 답변 감사합니다~ 구글 서치 콘솔에 등록 후 사이트 맵 설정 과정에서 문제를 겪고 있습니다. 혹시 sitemap.xml 파일을 따로 만들어 주어야하나요?
![image](https://user-images.githubusercontent.com/84106842/182024848-1c477083-9e48-40ff-b2bd-4804f1806db2.png)

**Reply by hyesungoh** [reply]
`robots.txt`, `sitemap.xml` 등 정적 파일 생성을 자동화해두어 따로 안만들어주셔도 됩니다. 

> `apps/blog/public`에 생성됩니다

https://dgd03146-github-io.vercel.app/sitemap.xml 접속해보니 문제가 없는 것으로 보아
현재 오류는 서치 콘솔에 등록한지 얼마 안되어 발생하는 오류로 추정됩니다.

**Reply by dgd03146** [reply]
정말 감사합니다!! 조금 기다려봐야겠어요. 혹시  블로그 이제까지 본것 중에 너무 깔끔하고 이쁩니다! 좋은 테마 만들어주셔서 감사합니다.
